client/signup/steps/plans/style.scss

```diff
@@ -1,6 +1,11 @@
 .plans-step {
 	margin: 0 auto;
 	max-width: 700px;
+
+	&.is-wide-layout {
```

Comment:
Yes. Instead of adding a new class name (e.g. `plans-features`), I add `is-wide-layout` if plans features is enabled. This way, I preserve the classnames rule (class name must be the same as folder name).

Do you think this is a bad idea?